runtime/lib/component/dispatcher.js

```diff
@@ -68,7 +69,7 @@ class Dispatcher {
       if (typeof name !== 'string') {
         return false
       }
-      var component = this.component[name]
+      var component = self.component[name]
       if (component == null) {
```

Comment:
Why use 'null', not 'undefined'?

Reply:
`==` is loose equality. `val == null` is true if val is null or undefined.